cpp/INIReader.cpp

```diff
@@ -68,6 +68,14 @@ bool INIReader::GetBoolean(const string& section, const string& name, bool defau
         return default_value;
 }
 
+// Return true if the given section exists
+bool INIReader::HasSection(const std::string& section) const
```

Comment:
Let's remove the `std::` prefix here as it's not needed with the `using` at the top (I realize `HasValue` has it, but I should remove that).

```suggestion
bool INIReader::HasSection(const string& section) const
```